scripts/install_libseccomp.sh

```diff
@@ -40,14 +41,37 @@ function build_and_install_libseccomp() {
     libseccomp_version=${opt_ver}
     libseccomp_url="https://github.com/seccomp/libseccomp"
     libseccomp_tarball="libseccomp-${libseccomp_version}.tar.gz"
-    libseccomp_tarball_url="${libseccomp_url}/releases/download/v${libseccomp_version}/${libseccomp_tarball}"
+
+    if [ ${libseccomp_version} = "main" ]; then
+        libseccomp_tarball_url="${libseccomp_url}/archive/refs/heads/${libseccomp_version}.tar.gz"
+    else
+        libseccomp_tarball_url="${libseccomp_url}/releases/download/v${libseccomp_version}/${libseccomp_tarball}"
+    fi
 
     echo "Build and install libseccomp version ${libseccomp_version}"
     libseccomp_install_dir=$opt_dir
     mkdir -p "${libseccomp_install_dir}"
-    curl -sLO "${libseccomp_tarball_url}"
+    curl -sL -o "${libseccomp_tarball}" "${libseccomp_tarball_url}"
     tar -xf "${libseccomp_tarball}"
     pushd "libseccomp-${libseccomp_version}"
+
+    if [ ${libseccomp_version} = "main" ]; then
+        ./autogen.sh
+        # Specify the tentative version of the libseccomp library because some
+        # functions of the Rust bindings are restricted based on the version.
+        sed -i "/^AC_INIT/ s/0.0.0/$TENTATIVE_MAIN_VER/" configure.ac
+        # Create a dummy pkgconfig because the Rust bindings need it for the compilation.
+        mkdir -p "${libseccomp_install_dir}/lib/pkgconfig"
+        cat << EOF > ${libseccomp_install_dir}/lib/pkgconfig/libseccomp.pc
```

Comment:
Maybe it's better to use `<<- EOF`? This allows to indent the here-document with tabs.

Reply:
@rusty-snake Thanks, 

In the first place, I didn't need to create the pkgconfig manually because it is created by `make install`, so I removed it.